azure-storage-blob/src/Blob/Internal/BlobResources.php

```diff
@@ -42,7 +42,7 @@ class BlobResources extends Resources
     // @codingStandardsIgnoreStart
 
     const BLOB_SDK_VERSION = '1.2.0';
-    const STORAGE_API_LATEST_VERSION = '2017-04-17';
+    const STORAGE_API_LATEST_VERSION = '2017-11-09';
```

Comment:
It needs to be evaluated if a skip in Storage API version (2017-07-29) is acceptable.

Reply:
I think this is alright. If anyone looks for an API version in between due to Azure Stack or some other problem, they could fallback to the earlier versions of the PHP SDK and use an older version API.